internal/handlers/common/params.go

```diff
@@ -121,7 +121,8 @@ var (
 	errNotBinaryMask         = fmt.Errorf("not a binary mask")
 	errUnexpectedLeftOpType  = fmt.Errorf("unexpected left operand type")
 	errUnexpectedRightOpType = fmt.Errorf("unexpected right operand type")
-	errLongExceeded          = fmt.Errorf("long exceeded")
+	errLongExceededPositive  = fmt.Errorf("long exceeded - positive value")
+	errLongExceededNegative  = fmt.Errorf("long exceeded - negative value")
```

Comment:
>  I added/updated comments and checked rendering.

> Before submitting a pull request, please make sure that:
> […]
> 2. Comments are added or updated for all new or changed code.
>    Please add missing comments for all (both exported and unexported)
>    new and changed top-level declarations (functions, types, etc).

Both points were ignored.

Updated possible error results are also not documented.

Reply:
@AlekSi could you please leave more specific feedback? What exactly needs to be documented here?
Sometimes when I leave comments (e.g. for structure fields) you say that they are obvious and not needed. What needs to be documented here?

Reply:
Should I document what each error in this list means?
Or should I document each function/method where these errors could be returned?
Or both? Or something else?